### PR TITLE
[NAE-1436] Create all views schematic

### PR DESCRIPTION
**Pull request by machacjozef**
# Description

<Please include a summary of the changes and which issue is fixed. Please also include relevant links and special instructions if applicable.>

Implements [NAE-1436]

## Dependencies

No new dependencies were introduced

### Third party dependencies

No new dependencies were introduced

### Blocking Pull requests

Depends on #94 #92 #60 

## How Has Been This Tested?

This was tested manually.

### Test Configuration

| Name                | Tested on |
|---------------------| --------- |
| OS                  |      Linux Mint 20.3     |
| Runtime             |  Node v12.22.12         |
| Dependency Manager  |   NPM 6.14.16    |
| Framework version   |    Angular 13.3.1     |
| Run parameters      |           |
| Other configuration |           |

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] My changes have been checked, personally or remotely, with @mladoniczky 
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have resolved all conflicts with the target branch of the PR
- [x] I have updated and synced my code with the target branch
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing tests pass locally with my changes:
    - [x] Lint test
    - [x] Unit tests
    - [x] Integration tests
- [x] I have checked my contribution with code analysis tools:
    - [x] [SonarCloud](https://sonarcloud.io/project/overview?id=netgrif_application-engine)
    - [x] [Snyk](https://app.snyk.io/org/netgrif)
- [x] I have made corresponding changes to the documentation:
    - [x] Developer documentation
    - [x] User Guides
    - [x] Migration Guides

[NAE-1436]: https://netgrif.atlassian.net/browse/NAE-1436?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: projects/nae-example-app/src/app/doc/public-single-task-view/public-single-task-view.component.html [hunk 1]
```diff
@@ -18,7 +18,7 @@
 
 <ng-template #pageHeader>
     <div>
-        <img src="../../../assets/img/netgrif_full_white.svg" alt="logo" class="logoimg">
+        <!-- <img src="TODO LINK" alt="logo" class="logoimg"> -->
```
Comment (renczesstefan) [inline projects/nae-example-app/src/app/doc/public-single-task-view/public-single-task-view.component.html:21]: This is only an example header image, so if this line is not used, the commented line can be removed.

File: projects/nae-example-app/src/app/doc/public-single-task-view/public-single-task-view.component.html [hunk 2]
```diff
@@ -28,6 +28,6 @@
 
 <ng-template #pageFooter>
     <mat-toolbar color="primary" class="footer-custom">
-        <span>NETGRIF, s.r.o. &copy; 2022</span>
+        <!-- TODO -->
```
Comment (renczesstefan) [inline projects/nae-example-app/src/app/doc/public-single-task-view/public-single-task-view.component.html:31]: Same as above

File: projects/netgrif-components/schematics/view/create-view-prompt/views/case-view/create-case-view.ts [hunk 3]
```diff
@@ -31,8 +31,8 @@ export function createCaseView(tree: Tree, args: CreateViewArguments & TabbedVie
         configName: projectInfo.projectNameClassified,
         configImportPath: createRelativePath(view.fileImportPath, `./${projectInfo.projectNameDasherized}-configuration.service`),
         caseCreationConfig: {
-            enableCaseTitle: args.enableCaseTitle,
-            isCaseTitleRequired: args.isCaseTitleRequired
+            enableCaseTitle: args.enableCaseTitle === undefined ? true : args.enableCaseTitle,
+            isCaseTitleRequired: args.isCaseTitleRequired === undefined ? false : args.isCaseTitleRequired
```
Comment (renczesstefan) [inline projects/netgrif-components/schematics/view/create-view-prompt/views/case-view/create-case-view.ts:35]: In the above two lines consider using nullish coalescing operator, like `args.enableCaseTitle ?? true`.

**Review by renczesstefan — CHANGES_REQUESTED** [review]

**Review by renczesstefan — DISMISSED** [review]

File: projects/netgrif-components/schematics/view/create-view-prompt/views/case-view/create-case-view.ts (hunk 3 above)
Comment (machacjozef) [reply projects/netgrif-components/schematics/view/create-view-prompt/views/case-view/create-case-view.ts:35]: Depends on https://github.com/netgrif/components/pull/94

File: projects/nae-example-app/src/app/doc/public-single-task-view/public-single-task-view.component.html (hunk 2 above)
Comment (machacjozef) [reply projects/nae-example-app/src/app/doc/public-single-task-view/public-single-task-view.component.html:31]: Depends on https://github.com/netgrif/components/pull/94

File: projects/nae-example-app/src/app/doc/public-single-task-view/public-single-task-view.component.html (hunk 1 above)
Comment (machacjozef) [reply projects/nae-example-app/src/app/doc/public-single-task-view/public-single-task-view.component.html:21]: Depends on https://github.com/netgrif/components/pull/94

File: projects/nae-example-app/src/app/doc/public-single-task-view/public-single-task-view.component.html (hunk 2 above)
Comment (Kovy95) [reply projects/nae-example-app/src/app/doc/public-single-task-view/public-single-task-view.component.html:31]: placeholder of footer should be there

**Review by mladoniczky — DISMISSED** [review]

**Review by mladoniczky — APPROVED** [review]

**Pull request merged**